app/assets/stylesheets/profile.css.scss

```diff
@@ -0,0 +1,7 @@
+html, head, body { margin: 0; padding: 0; width: 100%; height: 100% }
+
+
+body {
+  display: flex;
+  flex-flow: column;
+}
```

Comment:
Why is this here? Styling global elements from one page is probably not what you want.